Sources/Models/OATableDataModel.m

```diff
@@ -40,6 +40,24 @@ - (void)addSection:(OATableSectionData *)sectionData atIndex:(NSInteger)index
         [_sectionData insertObject:sectionData atIndex:index];
 }
 
+- (void)removeSection:(NSUInteger)section
+{
+    [_sectionData removeObjectAtIndex:section];
+}
+
+- (void)removeItemsAtIndexPaths:(NSArray<NSIndexPath *> *)indexPaths
+{
+    for (NSIndexPath *indexPath in indexPaths)
+    {
+        if (_sectionData.count > indexPath.section && [_sectionData[indexPath.section] rowCount] > indexPath.row)
+            [_sectionData[indexPath.section] removeRowAtIndex:indexPath.row];
+    }
+    for (NSIndexPath *indexPath in indexPaths)
+    {
+        if ([_sectionData[indexPath.section] rowCount] == 0)
```

Comment:
What if multiple index paths are in the same section? You need to check only once per given section